content/blog/2021-11-08-New-Visualisation-Page/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+author: [Dataviz Team]
+title: Introduce the new visualisation page
+description: We are excited to announce the new visualisation page in Dataviz.Shef!
+date: 2021-11-08
+category: [News]
+tag: [Dataviz, Research]
+thumbnail: visPage.png
+---
+
+Data visualisation is a great way to create impact, in support of the University’s vision, and supports the promotion of research. Looking back over the past year, data visualisation has been an effective tool for understanding worldwide trending topics such as climate change, CoVid-19, and elections. Attractive and understandable data visualisations would allow the university’s research to become more visible to different stakeholders. In addition, by sharing any source code, data, and other resources involved in making the visualisation, others will benefit from the openness and allowing them to easily reproduce or adapt from our materials, as well as drawing inspiration from it.
+
+In the past year, we have created the website <Link to="/">Dataviz.shef</Link>, hosted several <Link to="/events">events</Link>, and provided materials such as <Link to="/#learning-path">learning paths</Link>, <Link to="/blog">blog posts</Link>, and <Link to="/blog/category/tutorial">tutorials</Link> to a range of audiences in the university. Recently, inspired by [Beautiful News](https://informationisbeautiful.net/beautifulnews/), we have created a <Link to="/visualisation/">new visualisation page</Link> for latest research news stories at the University of Sheffield. Visualisations are categorised into departments/faculties and given relevant tags to help you find items which interest you the most.
+
+![New visualisation page](visPage.png)
+
+We are committed to making research news more understandable and accessible to wider audiences. Each visualisation comes with a brief description and several tabs containing information to help you digest and absorb the data. In addition, we have tested most of the visualisations using web accessibility tools to remove as many barriers as possible which prevent people accessing the latest news via data visualisation. From time to time you might want to explore some visualisations in detail. Since data and codes are open sourced (and digitally preserved) for almost every visualisation you would not have to worry about finding underlying resources.
```

Comment:
```suggestion
We are committed to making research news more understandable and accessible to wider audiences. Each visualisation comes with a brief description and several tabs containing information to help you digest and absorb the data. In addition, we have tested most of the visualisations using web accessibility tools to remove as many barriers as possible that prevent people from accessing the latest news via data visualisation. From time to time, you might want to explore some visualisations in detail. Since data and codes are open sourced (and digitally preserved) for almost every visualisation, you would not have to worry about finding underlying resources.
```